src/idl_gen_grpc.cpp

```diff
@@ -403,8 +403,14 @@ bool GeneratePythonGRPC(const Parser &parser, const std::string & /*path*/,
     namespace_dir += *it;
   }
 
-  std::string grpc_py_filename =
-      namespace_dir + kPathSeparator + file_name + "_grpc_fb.py";
+  std::string grpc_py_filename;
+  if (namespace_dir.empty()) {
+    grpc_py_filename = file_name + "_grpc_fb.py";
+  }
+  else{
```

Comment:
uses same style as surrounding code: `} else {`

Reply:
pull everything except `kPathSeparator` outside of the if-then-else

Reply:
made the changes.

Reply:
you didn't pull everything except `kPathSeparator` outside of the if-then-else..

Reply:
```
  -std::string grpc_py_filename =
      namespace_dir + kPathSeparator + file_name + "_grpc_fb.py";
  +std::string grpc_py_filename = file_name + "_grpc_fb.py";
  +if (!namespace_dir.empty()) {
  +  grpc_py_filename = namespace_dir + kPathSeparator + file_name + "_grpc_fb.py";
  +}
```
I had force pushed the above, are you commenting on the snippet above? not sure what to change.

Reply:
@aardappel  
 in this commit? https://github.com/google/flatbuffers/pull/5922/commits/ef60280a1309ceca20546ce7353659874376548a

Reply:
Yes, like I said, make sure the only thing inside the `if` is adding the `kPathSeparator `.

Reply:
updated in 7ca531f4

Reply:
Ok, let me write it out for you then:
```
auto grpc_py_filename = namespace_dir;
if (!namespace_dir.empty()) grpc_py_filename += kPathSeparator;
grpc_py_filename += file_name + "_grpc_fb.py";
```

Reply:
updated with the snippet above in 3787acb9961400b3640451882dea4115edbbab08 .

Reply:
@aardappel :point_up: